plugin/karma.c

```diff
@@ -228,6 +251,8 @@ static void karma_handler(cbot_event_t event, cbot_actions_t actions)
       actions.send(event.bot, event.channel, "sorry, you're not authorized to do that!");
     }
     recapfree(captures);
+  } else if (reexec(forget, event.message, NULL) != -1) {
```

Comment:
This will match any message that says "forget-me".  Really, we want to only match "*bot name* forget-me".  I use the `event.addressed()` function to tell if a message starts with a bot name (and optionally some punctuation) and whitespace.  You can see me using it in the cases above with the `increment` variable.  So overall, we would like the condition here to be `increment && reexec(forget, event.message + increment, NULL) != -1`.